### PR TITLE
CM-287: added payment method picker dropdown in payroll

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/CM-287

### CODE REVIEW

File: src/components/payroll/PaymentMethodPicker.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  errorPaymentMethods: state.payroll.errorPaymentMethods,
+  fetchedPaymentMethods: state.payroll.fetchedPaymentMethods,
+  paymentMethods: state.payroll.paymentMethods,
+});
+
+const mapDispatchToProps = (dispatch) => bindActionCreators({
+  fetchPaymentMethods,
+}, dispatch);
+
+export default injectIntl(connect(mapStateToProps, mapDispatchToProps)(PaymentMethodPicker));
```
Comment (olewandowski1) [inline src/components/payroll/PaymentMethodPicker.js]: Is it necessary to use _injectIntl_ here? You're providing label as a prop and it is already translated. So I think that _injectIntl_ could be removed from this Picker.

Comment (sniedzielski) [reply src/components/payroll/PaymentMethodPicker.js]: Sure. You are right. Let me check how it works without 'injectIntl'

**Review by olewandowski1 — CHANGES_REQUESTED** [review]
I've left one comment @sniedzielski .

File: src/components/payroll/PaymentMethodPicker.js (hunk 1 above)
Comment (olewandowski1) [reply src/components/payroll/PaymentMethodPicker.js]: It should not affect the component, since intl is not used "inside".

Comment (sniedzielski) [reply src/components/payroll/PaymentMethodPicker.js]: great, I've just applied your suggestion

**Comment by sniedzielski** [comment]
@olewandowski1 Please re-check PR. I changed base to develop as I decided to address comment from prev PR to develop related to payment status.

**Review by olewandowski1 — APPROVED** [review]
LGTM @sniedzielski

**Pull request merged**